src/lib/routes.tsx

```diff
@@ -3,6 +3,8 @@ export enum Routes {
   Dashboard = "/dashboard",
   Notification = "/notification",
   Attendance = "/attendance",
+  AddAttendance = "/attendance/add",
+  EditAttendance = "/attendance/edit?id=id",
```

Comment:
I think we should use `useParams` instead of relying on param queries for routes.

Reply:
Fixed